drivers/virtio/bus/include/virtio/virtio_bus.h

```diff
@@ -0,0 +1,376 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
How did the `virtio_bus.h` file come to be? It should have been moved, similar to `virtio_bus.c`.